doc/quickstart.rst

```diff
@@ -0,0 +1,186 @@
+Charm++ Quickstart
+==================
+
+This section gives a concise overview of running your first Charm++ application.
+
+Installing Charm++
+------------------
+
+To download the latest Charm++ release, run:
+
+.. code-block:: console
+
+   $ wget http://charm.cs.illinois.edu/distrib/charm-6.9.0.tar.gz
+   $ tar xzf charm-6.9.0.tar.gz
+
+To download the development version of Charm++, run:
+
+.. code-block:: console
+
+   $ git clone https://github.com/UIUC-PPL/charm
+
+
+To build Charm++, use the following commands:
+
+.. code-block:: console
+
+   $ cd charm
+   $ ./build AMPI netlrts-linux-x86_64 --with-production -j4
+
+This is the recommenced version to install Charm++ on Linux systems.
+For MacOS, substitute "linux" with "darwin". For advanced compilation options,
+please see Section :numref:`sec:install` of the manual.
+
+
+Parallel "Hello World" with Charm++
+-----------------------------------
+
+The basic unit of computation in Charm++ is a **chare**, which is a C++
+object. Chares have **entry methods** that can be invoked asynchronously.
+in A Charm++ application consists of collections of chares (such as chare arrays)
```

Comment:
In a Charm++

Reply:
Actually, looking at the sentence, there should be no "in". It should just be "A Charm++ application consists"

Reply:
fixed